### ISSUE TITLE
Visual Representation of similarities of contemplation cycles

### ISSUE THREAD

**Issue by katquatermass** [neontribe/contemplating-action#10]
The project plan has a word description of this - would be great if @HedgehogHolly  can do a visual one - showing how these cycles used in different types of situation overlap.  Here are google links to some of the image resources out there:

[[Drug and Alcohol]](https://www.google.com/search?biw=1368&bih=802&tbm=isch&sa=1&ei=jbxBXKSBJdOU1fAPreWi-A4&q=recovery+journey+contemplation+cycle&oq=recovery+journey+contemplation+cycle&gs_l=img.3...3942.8475..8695...0.0..0.72.1157.20......1....1..gws-wiz-img.......0j0i30j0i24.y5_8bbnFlR0#imgrc=_)

[Page 13 of this](https://docs.wixstatic.com/ugd/f86f13_366b6514c8fc4e9488fc15edf2148d52.pdf)

[The green table on this page](https://www.change4health.org/it-begins-with-you/recovery-resilience/ibwu-recovery/)

**Comment by katjam** [comment]
@time of writing - Clipped from project plan for reference

ContemplatingAction starts from the idea that people who need to make a change in their lives are often on a similar overall journey which travels broadly along the following lines:

pre-contemplation/unaware   contemplation   seeking help  taken action     seeking to maintain    recovery

This journey is not fully linear, once the unaware stage has been passed people may cycle through the interim steps many times before reaching a more stable recovery

We believe that helping people transition through “contemplation” into “being actively ready to seek help” is a goal that tech can help charities achieve.

Specifically in this project we want to test one particular pathway to see if it can be effective in helping people with different problems make this step:

1) Individual is in their own thoughts, but engaged with a social media channel as a “downtime” activity. Fears and barriers are holding them back from actively seeking help.

2) An intervention in that channel takes them to a small app

3) That app tackles those fears by a combination of short stories with illustrations demonstrating positive outcomes and short information passages in plain english

4) The individual leaves that app by one of the routes it offers to real time support (a helpline or chatline)

**Comment by katquatermass** [comment]
Yes - this is a @HedgehogHolly lowish priority task but one that would be very useful for us when communicating about the project.  Does it make enough sense Holly?

**Comment by HedgehogHolly** [comment]
I've mocked up something for you. Im not sure if you wanted it in a circular pattern? I can easily change that if so. Let me know if it is what you have in mind.

![pathway-100](https://user-images.githubusercontent.com/41140803/51920649-bef54380-23dd-11e9-9f3d-575f74a2dec3.jpg)
[Google Drive File](https://drive.google.com/file/d/1s4iPqzwfKQIXhmfKOV822Kt1jk6uGYD0/view?usp=sharing)

**Comment by katquatermass** [comment]
Holly that's great and doesn't need to be circular at all (except your app icon looks a bit like a book to me!).  So that one is the flow of our steps.  The other one I am looking for is the bit at the beginning - I want to express visually how many different types of recovery journey are similar... Might need us to chat a bit first?

**Comment by HedgehogHolly** [comment]
Thats because it is a book 😆 The book was in reference to the stories on the app. However, I can change it if you don't think it is appropriate.

I think it would be helpful to have a chat about the beginning bit whenever you have a moment.

**Comment by katjam** [comment]
Would it be helpful to illustrate that we are not expecting that first visit to swing them necessarily. so the final step is either they get in touch or they go back to life and something else triggers them to come back one or more times and at some point they get in touch? I thought that was one of the aims of the concept to comfort them a step at a time and not overwhelm them. 

Or is the the 'circular' version that Kat thinks might not be necessary?

**Comment by katquatermass** [comment]
@roseb04 to do v short copy for the different Theory of Change type cycles available in these different areas:
Drugs and Alcohol
Abuse (Tech for Abuse report)
Living with Illness
Mental Health

And then v short copy for the sections in a core ToC cycle

@HedgehogHolly to then arrange to display how multiple different "supporting people" charities are all actually using the same model

NB. Style that goes well with our journey representation that Holly has already done also very useful

**Comment by katquatermass** [comment]
> Would it be helpful to illustrate that we are not expecting that first visit to swing them necessarily. so the final step is either they get in touch or they go back to life and something else triggers them to come back one or more times and at some point they get in touch? I thought that was one of the aims of the concept to comfort them a step at a time and not overwhelm them.


Oh! Actually that's great Katja - it does need to show that - the final visit is the measurable one but between button 3 and the final step we want both the direct pathway as shown AND the pathway that allows shows people leaving the app getting on with life and then returning to it.

I think it looks like two extra boxes on a line below from Box 3  "Individual has read enough for now, and closes the app to get on with other things"   "An incident occurs which reminds the individual of the pressures of their situation so they decide to look at the app again"   - the triangle of lines connecting need arrows of direction on them - and possibly a shadow or dotted arrow line/arrow combo to suggest that they go round multiple times.   Yell if that doesn't make sense @HedgehogHolly 

(The cyclic thing actually came from something else which was a different visual representation idea, we need both)

**Comment by rosejbon** [comment]
I'm working on the copy here, let me know if this ok: https://docs.google.com/document/d/1OgXJllFoJdmSf6bWVmLFyMc9maqZmxkaID8e-iHkrcE/edit?usp=sharing

I quite like this visual, it may be a bit fluid. But it shows that there is fluidity in the model and people can go through the cycle, enter at different points, skip stages etc. ![untitled_clipping_013119_051542_pm](https://user-images.githubusercontent.com/44296500/52073314-1c74c600-257f-11e9-9834-d788cbf865b0.jpg)

**Comment by katquatermass** [comment]
@HedgehogHolly have left a bundle of comments on Rose's google doc above about what we want to get to and how we intend to use it. IF it makes sense, you can maybe go ahead with next iterations. If it doesn't don't worry - we'll talk about it at the 3pm call or just after

**Comment by HedgehogHolly** [comment]
![pathway-100](https://user-images.githubusercontent.com/41140803/52405836-29843e80-2ac4-11e9-96d7-a42cb8272437.jpg)

Is this more what you were thinking of @katquatermass and @katjam?

**Comment by katquatermass** [comment]
So look again at the cycle - if you read the current flow it doesn't make sense.
the right hand top line box is an end point.
The left hand bottom line box is another temporary end point, which might then flow to incident and then from incident they would go back into the app - not direct to phone line.

So fork at end of top line two diagonals - phone on top door on bottom
Then to the left off door the incident - with an arrow for direction of travel to it and then back up to app
(doesn't have to be that exact layout - triangle of repeatable route (that needs arrows) could be displayed elsehow

**Comment by katquatermass** [comment]
Sorry - I'd draw on it if I could! - but the phrasings only make sense in one set of cycles - I think!

**Comment by HedgehogHolly** [comment]
Ah yes, I see what you mean.  So it would look more like this? 
![pathway-100](https://user-images.githubusercontent.com/41140803/52415860-167f6780-2ae0-11e9-9c09-76aab520a8ee.jpg)

**Comment by katquatermass** [comment]
Add a few more arrows and yes, that's it I think
Kat
